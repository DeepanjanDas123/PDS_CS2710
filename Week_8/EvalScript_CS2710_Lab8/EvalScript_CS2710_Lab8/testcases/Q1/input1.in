3
5
push_back 5
push_back 6
push_back 7
push_back 8
pop_front