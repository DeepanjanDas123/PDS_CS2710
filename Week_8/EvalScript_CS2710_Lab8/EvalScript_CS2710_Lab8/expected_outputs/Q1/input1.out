Deque is full
5